tests/integration_app_test.py

```diff
@@ -1,59 +1,40 @@
 import os
 import pytest
 from dotenv import load_dotenv, find_dotenv
-import requests
+import pymongo
+import mongomock
+from bson.objectid import ObjectId
 from todo_app import app
 
 
 @pytest.fixture
 def client():
-    # Use test environment config here
-    file_path = find_dotenv('.env.test')
-    load_dotenv(file_path, override=True)
+    filepath = find_dotenv(".env.test")
+    load_dotenv(filepath, override=True)
 
-    # Create new app instance
-    test_app = app.create_app()
+    with mongomock.patch(servers=(('fakemongo.com', 27017),)):
+        test_app = app.create_app()
+        with test_app.test_client() as client:
+            yield client
 
-    # Create client using the new app instance
-    with test_app.test_client() as client:
-        yield client
 
+def test_index_page(client):
+    get_mock_mongo_collection()
+    response = client.get('/')
 
-class StubResponse():
-    def __init__(self, fake_response_data):
-        self.fake_response_data = fake_response_data
-        self.status_code = 200
-
-    def json(self):
-        return self.fake_response_data
-
-
-def get_mock_trello(url, params={}):
-    BOARD_ID = os.environ.get('BOARD_ID')
-    # The expected URL should match the URL format used in your application
-    expected_url = f'https://api.trello.com/1/boards/{BOARD_ID}/lists'
-
-    if url == expected_url:
-        return get_mock_trello_cards()
-
-    raise Exception(f'Integration test did not expect URL "{url}"')
-
+    html_response = response.data.decode()
 
-def get_mock_trello_cards():
-    fake_response_data = [{
-        'id': '123abc',
-        'name': 'To do',
-        'cards': [{'id': '456', 'name': 'Test card'}, {'id': '459', 'name': 'Mock data for tests'}]
-    }]
-    return StubResponse(fake_response_data)
+    assert response.status_code == 200
+    assert 'Test todo (From integration test)' in html_response
 
 
-def test_index_page(monkeypatch, client):
-    monkeypatch.setattr(requests, 'get', get_mock_trello)
+def get_mock_mongo_collection():
```

Comment:
Minor, but would this method be better described as `add_item_to_db` or something similar? It does involve getting the collection, but it doesn't return the collection and that's not really the _point_ of the method, more a thing it happens to do